include/odp/api/spec/ipsec.h

```diff
@@ -762,6 +770,38 @@ typedef struct odp_ipsec_sa_param_t {
 
 } odp_ipsec_sa_param_t;
 
+/**
+ * IPSEC stats content
+ */
+typedef struct odp_ipsec_stats_t {
+	/** Number of packets processed successfully */
+	uint64_t success;
+
+	/** Number of packets with protocol errors */
+	uint64_t proto_err;
+
+	/** Number of packets with SA lookup failures */
+	uint64_t sa_lookup_err;
```

Comment:
@JannePeltonen I think we should remove sa_lookup_err from this list. Reason, we are collecting stats per SA. An SA lookup error cannot be associated to an SA. Can you confirm I can make this change?

Reply:
Yes, of course. Obviously lookup failure cannot be in per-SA stats and somehow we missed it.